### PR TITLE
Implementation of search on contents

### LINKED ISSUE #131: Design and Implement Search on contents
On the Course consumption screen, it will be handy to be able to search on content names. 

- [x] Design and layout of the search UI (Bounty: 200)
- [ ] Implementation for Design (CSS, to done via motley) (Bounty: 500)
- [ ] Feature Implementation (Ember) (Bounty: 1000)

Note: No need to make separate API calls to get search results, assume all data to be loaded on the frontend only.

This is the course consumption screen:

screen shot 2018-06-19 at 8 12 04 pm

### PR DESCRIPTION
**Pull request by architkshk**
fixes #131 @abhishek97 
to be merged after https://github.com/coding-blocks/motley/pull/232

### CODE REVIEW

File: app/pods/components/content-search/component.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  qs = ''
+  @alias('searchTask.lastSuccessful.value') results
+
+  @service router
+  @service store
+
+  searchTask = task(function*() {
+    yield timeout(1000);
+    const searchQuery = this.get("qs");
+    const contents = this.get("store").peekAll("content");
```
Comment (abhishek97) [inline app/pods/components/content-search/component.js]: no this will list ALL of the contents in store, we only want those related to `this` runAttempt

**Comment by architkshk** [comment]
@abhishek97 done! pls review.

File: app/models/content.js [hunk 2]
```diff
@@ -25,5 +25,5 @@ export default DS.Model.extend({
     }
   }),
   progress: DS.belongsTo('progress'),
-  //section: DS.belongsTo('section')
+  section: DS.belongsTo('section')
```
Comment (abhishek97) [inline app/models/content.js]: This shall remain commented for the simple reason, a content may belong to many sections at a time.

Comment (architkshk) [reply app/models/content.js]: Done @abhishek97

**Comment by architkshk** [comment]
@abhishek97  Done the suggested changes.

**Pull request closed without merging**